interfaces/login_ui_endpoints/v0/schema.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2023 Canonical
+# See LICENSE file for licensing details.
+"""This file defines the schemas for the provider and requirer sides of the login_ui_endpoints interface.
+It exposes two interfaces.schema_base.DataBagSchema subclasses called:
+- ProviderSchema
+- RequirerSchema
+Examples:
+    ProviderSchema:
+        unit: <empty>
+        app: {"consent_url": "http://identity-platform-login-ui-url/consent",
+              "error_url": "http://identity-platform-login-ui-url/error",
+              "index_url": "http://identity-platform-login-ui-url/index",
+              "login_url": "http://identity-platform-login-ui-url/login",
+              "oidc_error_url": "http://identity-platform-login-ui-url/oidc_error",
+              "registration_url": "http://identity-platform-login-ui-url/registration",
+              "default_url": "http://identity-platform-login-ui-url"
+              }
+"""
+
+from pydantic import BaseModel
+
+from interfaces.schema_base import DataBagSchema
```

Comment:
a PR just merged that moved this file out of this repo. 
```suggestion
from interface_tester.schema_base import DataBagSchema
```
If you're developing locally you can pip install https://github.com/PietroPasotti/interface-tester-pytest (temporary, will be in pypi soon)

Reply:
done